modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/cloud/aws/AWS.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2012-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+package com.snowplowanalytics.snowplow.rdbloader.common.cloud.aws
+
+import cats.effect._
+import cats.implicits._
+import com.snowplowanalytics.snowplow.rdbloader.common.cloud.BlobStorage.Key
+import fs2.{Pipe, Stream}
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.s3.S3AsyncClient
+import org.typelevel.log4cats.Logger
+import blobstore.s3.{S3Path, S3Store}
+import scala.concurrent.duration._
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder
+import com.amazonaws.services.simplesystemsmanagement.model.{GetParameterRequest, AWSSimpleSystemsManagementException}
+
+// This project
+import com.snowplowanalytics.snowplow.rdbloader.common.cloud.{BlobStorage, Queue}
+
+object AWS {
```

Comment:
It might benefit from organizing functions a bit further. i.e. object AWS { object AwsQueue{}, AwsAuth{}, etc.}. Same for GCP.scala